packages/livechat/src/widget.ts

```diff
@@ -489,20 +489,13 @@ const api: InternalWidgetAPI = {
 		if (!config.url) {
 			throw new Error('Config.url is not set!');
 		}
+		const urlToken = token && `&token=${token}`;
+
 		api.popup = window.open(
-			`${config.url}${config.url.lastIndexOf('?') > -1 ? '&' : '?'}mode=popout`,
+			`${config.url}${config.url.lastIndexOf('?') > -1 ? '&' : '?'}mode=popout${urlToken}`,
```

Comment:
If token is false or undefined this can cause problems like having the explicit `undefined` value in the URL.

Reply:
When the widget is initialized a token is always generated, so it never is undefined

Reply:
If its always defined, why are we checking it if its not undefined? 👀